update/Pos/Update/Genesis.hs

```diff
@@ -0,0 +1,22 @@
+module Pos.Update.Genesis
+       ( UpdateGenesis(..)
+       ) where
+
+import           Data.Time.Units            (Millisecond)
+import           Serokell.Data.Memory.Units (Byte)
+
+import           Pos.Core.Types             (BlockVersion, ScriptVersion, SoftwareVersion)
+import           Pos.Update.Core.Types      (BlockVersionData)
+
+-- | Genesis data needed for the update system. It's defined here but
+-- provided in cardano-sl (specifically, "Pos.Genesis"). Modules in
+-- cardano-sl can access these fields as constants, but modules in
+-- cardano-sl-update have to grab them from context using 'askContext'.
```

Comment:
Btw, why not define them in `cardano-sl-update` as constants? File with constants is in `cardano-sl-core` and IIRC we have some hierarchical configs and constants.